js/main.js

```diff
@@ -20,14 +30,82 @@ function handleSubmit(event) {
   event.preventDefault();
   const newFormEntry = {
     entryId: data.nextEntryId,
-    nextEntryId: data.nextEntryId + 1,
     title: $entryTitle.value,
     photoURL: $photoURL.value,
     notes: $entryNotes.value,
   };
   data.nextEntryId += 1;
   data.entries.unshift(newFormEntry);
+  if (data.entries.length > 0) {
+    toggleNoEntries();
+  }
+  window.location.reload();
```

Comment:
You should not be forcing the page to reload. That's the whole purpose of using `event.preventDefault()`